recipe/meta.yaml

```diff
@@ -16,8 +16,12 @@ build:
 requirements:
   build:
     - {{ stdlib('c') }}
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
+    # For Linux CUDA 11.8 builds, pin to GCC 10 to workaround issues with GCC 11.
+    # xref: https://github.com/conda-forge/cudadecon-feedstock/issues/29
+    - {{ compiler('c') }}                          # [not (linux and cuda_compiler_version == "11.8")]
+    - {{ compiler('cxx') }}                        # [not (linux and cuda_compiler_version == "11.8")]
+    - {{ c_compiler }}_{{ target_platform }} 10    # [linux and cuda_compiler_version == "11.8"]
+    - {{ cxx_compiler }}_{{ target_platform }} 10  # [linux and cuda_compiler_version == "11.8"]
```

Comment:
This is a workaround to pin to GCC 10 as it was seen to work before with other builds (like CUDA 11.2). So hopefully this also works with CUDA 11.8

To track removing this workaround, filed issue: https://github.com/conda-forge/cudadecon-feedstock/issues/29